flink-kubernetes-webhook/src/main/java/org/apache/flink/kubernetes/operator/admission/FlinkOperatorWebhook.java

```diff
@@ -57,8 +59,8 @@ public class FlinkOperatorWebhook {
     public static void main(String[] args) throws Exception {
         EnvUtils.logEnvironmentInfo(LOG, "Flink Kubernetes Webhook", args);
         FlinkConfigManager configManager = new FlinkConfigManager();
-        AdmissionHandler endpoint =
-                new AdmissionHandler(new FlinkValidator(new DefaultValidator(configManager)));
+        Set<FlinkResourceValidator> validators = ValidatorUtils.discoverValidators(configManager);
```

Comment:
It needs to add `FLINK_PLUGINS_DIR` env variable into `webhook.yaml`, WDYT?

Reply:
The env `FLINK_PLUGINS_DIR` has been set in the `flink-operator.yaml` for webhook container